### ISSUE TITLE
savePLYFile does not save face elements

### ISSUE THREAD

**Issue by VictorLamoine** [PointCloudLibrary/pcl#1470]
When saving a point cloud using [`savePLYFile`](http://docs.pointclouds.org/trunk/group__io.html#gac5383ee048888dccb9a6d302ee5dea00); the number of faces (always 0) is not written in the file.

The file is ok to be read by MeshLab etc.. but this missing line prevents loading the file with [`loadPolygonFilePLY`](http://docs.pointclouds.org/trunk/group__io.html#ga8080e931aa4436bdad8220bddb25df35) because VTK expects a `face 0`.

Here is the minimum expected by VTK:

``` cpp
  if ( (elem = vtkPLY::find_element (ply, "vertex")) == NULL ||
       vtkPLY::find_property (elem, "x", &index) == NULL ||
       vtkPLY::find_property (elem, "y", &index) == NULL ||
       vtkPLY::find_property (elem, "z", &index) == NULL ||
       (elem = vtkPLY::find_element (ply, "face")) == NULL ||
       vtkPLY::find_property (elem, "vertex_indices", &index) == NULL )
    {
    vtkErrorMacro(<<"Cannot read geometry");
    vtkPLY::ply_close (ply);
    }
```

**Comment by kunaltyagi** [comment]
Would merely adding an extra "face 0" line to the PLY writer suffice?

**Comment by VictorLamoine** [comment]
From what I remember yes!

**Comment by mvieth** [comment]
Fixed by #4774
